### PR TITLE
Made a page for Bots

### PR DESCRIPTION
**Pull request by lealgamb**
Based on the feedback we got some people recommended a page with information about the bots we offer so here it is. 

image

### CODE REVIEW

File: src/main/java/codeu/controller/BotServlet.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet class responsible for the bots page. */
+public class BotServlet extends HttpServlet {
+
+  /** Controller class that gives control to Bots. */
+  private BotController botController;
+
+  /** Set up state for handling activty feed requests. */
```
Comment (yebrahim) [inline src/main/java/codeu/controller/BotServlet.java]: Doc.

File: src/main/webapp/WEB-INF/view/bots.jsp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<head>
+  <title>Bots</title>
+  <%@ include file="navbar.jsp" %>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <div id="container">
+    <h1>Bots
+      <a href="" style="float: right">&#8635;</a></h1>
```
Comment (yebrahim) [inline src/main/webapp/WEB-INF/view/bots.jsp]: What does this accomplish?

File: src/main/webapp/css/main.css [hunk 3]
```diff
@@ -129,11 +129,9 @@ button {
   overflow-y: scroll;
 }
 
-/* This is for the Search Results Page */
-#results {
```
Comment (yebrahim) [inline src/main/webapp/css/main.css]: We not using this anymore or removed by mistake?

Comment (lealgamb) [reply src/main/webapp/css/main.css]: Not using it anymore. It was from when the search bar redirected to a results page.

File: src/main/webapp/WEB-INF/view/bots.jsp (hunk 2 above)
Comment (lealgamb) [reply src/main/webapp/WEB-INF/view/bots.jsp]: It was to update the bots, but I now realize this only happens when the site is updated. I'll remove it.

File: src/main/java/codeu/controller/BotServlet.java (hunk 1 above)
Comment (lealgamb) [reply src/main/java/codeu/controller/BotServlet.java]: Oops, sorry about that.

**Pull request merged**